src/index.ts

```diff
@@ -12,8 +12,8 @@ await heliaServer.isReady
 // Add the prometheus middleware
 const app = Fastify({
   logger: {
-    enabled: DEBUG !== '',
-    msgPrefix: 'helia-http-gateway:fastify',
+    enabled: FASTIFY_DEBUG !== '',
+    msgPrefix: 'helia-http-gateway:fastify ',
```

Comment:
not having a trailing space caused logs prefixed with `helia-http-gateway:fastifyincoming` instead of `helia-http-gateway:fastify incoming`